src/main/scala/net/cakesolutions/CakeDynVerPlugin.scala

```diff
@@ -30,22 +30,30 @@ object CakeDynVerPlugin extends AutoPlugin {
     * set, eval, and .sbt files.
     */
   object autoImport {
+
```

Comment:
Lets fill the empty string below with a description

Reply:
done